### DISCUSSION TITLE
25th open maintainers call

### DISCUSSION THREAD

**Discussion by thomaseizinger** [libp2p/rust-libp2p#4209] [General]
Hi everyone,

The rust-libp2p maintainers will meet tomorrow (Tuesday) 09:00 UTC. It looks like it was removed from the public calendar, probably because @mxinden is on vacation. I am gonna be there anyway in case anyone has any questions.

https://meet.jit.si/rust-libp2p-open-maintainers-call

Everyone is welcome to join, listen along, propose topics and join the discussions.

Please post below in case you already want to add an agenda item.

See you all tomorrow!

**Comment by scholz-timo** [comment]
Hey, sadly I missed the call, is there any possibility to schedule a 1o1 call or to mark me on the next rust-libp2p maintainers call?

**Reply by thomaseizinger** [reply]
Both Max and I are on low-availability this and next week but we do these regularly (every 2 weeks) so if is not super urgent, I'd say just mark the next one (August 1st) in your calendar or subscribe to the public calendar directly: https://calendar.google.com/calendar/embed?src=libp2p.io_0q9682i3te7eanhe9q7ae1c58g%40group.calendar.google.com

**Comment by mxinden** [comment]
@thomaseizinger we moved this week's call to tomorrow, Thursday, 9 am UTC.

Hope this still works for you. Thank you for doing an additional call yesterday.

//CC @scholz-timo since this is another opportunity for you to join.

**Reply by thomaseizinger** [reply]
> @thomaseizinger we moved this week's call to tomorrow, Thursday, 9 am UTC.

Ha, I noticed too when my phone reminded me today :sweat_smile:

**Comment by thomaseizinger** [comment]
@b-zee We have this call happening in ~40 minutes in case you wanted to have a sync discussion on https://github.com/libp2p/rust-libp2p/pull/4189 :)
